### PR TITLE
[IT-2661] Use Synapse PAT instead of api token

### PR DESCRIPTION
**Pull request by zaro0508**
Switch to using a Synapse PAT to authenticate with Synapse. This is the recommended way to authenticate going forward.

### CODE REVIEW

**Review by brucehoff — APPROVED** [review]
Looks great!

**Review by BryanFauble — APPROVED** [review]
Changes LGTM!

File: GenerateAndSendStorageReport.py [hunk 1]
```diff
@@ -13,9 +13,9 @@
 currentDate = datetime.datetime.now().isoformat()[:10]  # YYYY-MM-DD
 currentTime = datetime.datetime.now().isoformat()[:19]
 
-def createAndSendReport(email, apiKey, recipientIds, tableId):
+def createAndSendReport(email, token, recipientIds, tableId):
```
Comment (thomasyu888) [inline GenerateAndSendStorageReport.py:16]: Nit: you can technically remove email all together as well, but its fine to keep it.

**Review by thomasyu888 — APPROVED** [review]

**Review by xschildw — APPROVED** [review]

File: GenerateAndSendStorageReport.py (hunk 1 above)
Comment (zaro0508) [reply GenerateAndSendStorageReport.py:16]: i'll let the original author, @nickgros, decide there.

Comment (nickgros) [reply GenerateAndSendStorageReport.py:16]: Agree with @thomasyu888

**Pull request merged**